src/services/tx/tx-sender/sdk.ts

```diff
@@ -126,8 +123,9 @@ export const getAssertedChainSigner = async (
  * most of the values of transactionResponse which is needed when
  * dealing with smart-contract wallet owners
  */
-export const getUncheckedSafeSDK = async (onboard: OnboardAPI, chainId: SafeInfo['chainId']): Promise<Safe> => {
-  const signer = await getAssertedChainSigner(onboard, chainId)
+export const getUncheckedSafeSDK = async (provider: Eip1193Provider): Promise<Safe> => {
+  const browserProvider = createWeb3(provider)
+  const signer = await browserProvider.getSigner()
```

Comment:
If we update to the latest protocol-kit it actually expects EIP 1193 Providers and removes all adapters.
Should we use the opportunity to do this in this PR?

https://docs.safe.global/sdk/protocol-kit/reference/migrate-to-v4

Reply:
Thats really nice! Doesn't look like too big of a change but just to be safe I would do the migration in a follow-up PR after this is merged.